cirq/ops/pauli_string.py

```diff
@@ -652,8 +652,8 @@ def zip_paulis(self, other: 'cirq.PauliString'
                   ) -> Iterator[Tuple[pauli_gates.Pauli, pauli_gates.Pauli]]:
         return (paulis for qubit, paulis in self.zip_items(other))
 
-    def _commutes_(self, other: Any,
-                   atol: float) -> Union[bool, NotImplementedType, None]:
+    def _commutes_(self, other: Any, *, atol: Union[int, float] = 1e-8
```

Comment:
Why the addition of ` 1e-8`?

Reply:
I've just added a default value. Across Cirq code there are sometimes `_commutes_` methods with the default value and sometimes without, e.g.:
- cirq/ops/raw_types.py:Gate -> `def _commutes_(self, other: Any, atol: float) -> Union[None, NotImplementedType, bool]:`
- cirq/ops/raw_types.py:TaggedOperation -> `def _commutes_(self, other: Any, *, atol: Union[int, float] = 1e-8) -> Union[bool, NotImplementedType, None]:`

What are the advantages of removing the default value?